imap_processing/tests/swapi/test_swapi_l2.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+
+from imap_processing import imap_module_directory
+from imap_processing.cdf.utils import write_cdf
+from imap_processing.swapi.l1.swapi_l1 import swapi_l1
+from imap_processing.swapi.l2.swapi_l2 import TIME_PER_BIN, swapi_l2
+
+
+def test_swapi_l2_cdf():
+    """Test housekeeping processing and CDF file creation"""
+    l0_data_path = (
+        f"{imap_module_directory}/tests/swapi/l0_data/"
+        "imap_swapi_l0_raw_20231012_v001.pkts"
+    )
+    processed_data = swapi_l1(l0_data_path, data_version="v001")
+    l1_dataset = processed_data[0]
+
+    l2_dataset = swapi_l2(l1_dataset, data_version="v001")
```

Comment:
It is worth thinking about if the input `l1_dataset` could be synthetic or come from an l1 file. At some point, it is nice to not need to run the l1 processing to generate test data for l2. That is up to you as long as you keep in mind that avoiding adding tests that take a long time to run is good.

Reply:
I will think about it. May not do that in this PR but will keep in mind.